docker-for-windows/troubleshoot.md

```diff
@@ -464,19 +447,12 @@ For more information, see:
 * [Windows NAT capabilities and
   limitations](https://blogs.technet.microsoft.com/virtualization/2016/05/25/windows-nat-winnat-capabilities-and-limitations/)
 
+### Running Docker Desktop in nested virtualization scenarios
 
-### Running Docker Desktop for Windows in nested virtualization scenarios
-
-Docker Desktop for Windows can run inside a Windows 10 virtual machine (VM) running on
-apps like Parallels or VMware Fusion on a Mac provided that the VM is properly
-configured. However, problems and intermittent failures may still occur due to
-the way these apps virtualize the hardware. For these reasons, _**Docker Desktop for
-Windows is not supported for nested virtualization scenarios**_. It might work
+Docker Desktop can run inside a Windows 10 virtual machine (VM) running on apps like Parallels or VMware Fusion on a Mac provided that the VM is properly configured. However, problems and intermittent failures may still occur due to the way these apps virtualize the hardware. For these reasons, _**Docker Desktop is not supported in nested virtualization scenarios**_. It might work
 in some cases, and not in others.
 
-The better solution is to run Docker Desktop for Windows natively on a Windows system
-(to work with Windows or Linux containers), or Docker Desktop for Mac on Mac to work
-with Linux containers.
+The better solution is to run Docker Desktop natively on a Windows system (to work with Windows or Linux containers), or on Mac to work with Linux containers.
```

Comment:
nit: is this not the documentation for Windows? Should we mention the Mac there?

Reply:
I think there is a reference to Mac here for completeness. I checked the Mac topic on logs and troubleshooting, but couldn't find any info about the nested virt scenario. Therefore, I'm tempted to leave this one 'as is'. Let me know if you disagree.